### ISSUE TITLE
could not determine kind of name for C.gtk_menu_popup_at_pointer (Part 2)

### ISSUE THREAD

**Issue by ei8fdb** [nogoegst/onionize#51]
Hea @nogoegst,

As mentioned in #44, I've still got this issue: I can't get onionize running.

OK, I've removed Golang which was installed from standard repositories, and replaced it with 1.9.3, but I still have the same issue as above.

**OS**
same as above

**Golang version**
```
bernard@MACHINE:~$ go version
go version go1.9.3 linux/amd64
```

Attempt to install onionize
```
bernard@MACHINE:~$ go get -u -tags gui github.com/nogoegst/onionize/cmd/onionize
# github.com/nogoegst/onionize/vendor/github.com/gotk3/gotk3/gtk
go/src/github.com/nogoegst/onionize/vendor/github.com/gotk3/gotk3/gtk/
menu_since_3_22.go:18:2: could not determine kind of name for 
C.gtk_menu_popup_at_pointer
```

This is the same issue as I had previously above.

Any suggestions? Thanks.

**Comment by unkaktus** [comment]
It seems that there are somehow broken/outdated GTK3+ dependencies. So I will fire up some Ubuntu/Debian to add required dependencies to the documentation.

I should really go with binary releases (#47) because reproducing build environment for non-Go code in a pain (GTK3+, especially on Windows and macOS). I am desperately looking for some really good platform-independent UI solution... #53.

In meantime you can try to figure out your GTK3+ version and pass it to build command (if I recall correctly)...
```
go get -u -tags "gui,gtk_3_10" github.com/nogoegst/onionize/cmd/onionize
```

Thank you for the report!

**Comment by unkaktus** [comment]
Hi again @ei8fdb,

I've spinned up an Ubuntu container and figured out that Xenial uses GTK 3.18. So the working command to build on Xenial is
```
go get -u -tags "gtk_3_18 gui" github.com/nogoegst/onionize/cmd/onionize
```

I will add Dockerfiles soon. You'll probably want to build with one because it is hell a lot of dependencies.

**Comment by unkaktus** [comment]
Should be solved by #58.
